### PR TITLE
18 Duskmourn cards + 2 Schemes

### PR DESCRIPTION
**Pull request by Fulgur14**

### CODE REVIEW

File: forge-gui/res/cardsfolder/upcoming/kaito_bane_of_nightmares.txt [hunk 1]
```diff
@@ -0,0 +1,13 @@
+Name:Kaito, Bane of Nightmares
+ManaCost:2 U B
+Types:Legendary Planeswalker Kaito
+Loyalty:4
+K:Ninjutsu:1 U B
+S:Mode$ Continuous | Affected$ Permanent.Self+counters_GE1_LOYALTY | Condition$ PlayerTurn | AddType$ Creature & Ninja | SetPower$ 3 | SetToughness$ 4 | AddKeyword$ Hexproof | Description$ During your turn, as long as Kaito has one or more loyalty counters on him, he's a 3/4 Ninja creature and has hexproof.
+A:AB$ Effect | Cost$ AddCounter<1/LOYALTY> | Name$ Emblem — Kaito, Bane of Nightmares | Image$ emblem_kaito_bane_of_nightmares | StaticAbilities$ STNinjaBoost | Planeswalker$ True | Duration$ Permanent | SpellDescription$ [+1]: You get an emblem with "Ninjas you control get +1/+1."
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/kaito_bane_of_nightmares.txt:7]: Bad description

File: forge-gui/res/cardsfolder/upcoming/overlord_of_the_mistmoors.txt [hunk 2]
```diff
@@ -0,0 +1,10 @@
+Name:Overlord of the Mistmoors
+ManaCost:5 W W
+Types:Enchantment Creature Avatar Horror
+PT:6/6
+K:Impending:4:2 W W
+T:Mode$ ChangesZone | Origin$ Any | Destination$ Battlefield | ValidCard$ Card.Self | Execute$ TrigToken | TriggerDescription$ Whenever CARDNAME enters or attacks, create two 2/1 white Insect creature tokens with flying.
+T:Mode$ Attacks | ValidCard$ Card.Self | TriggerZones$ Battlefield | Secondary$ True | Execute$ TrigToken | TriggerDescription$ Whenever CARDNAME enters or attacks, create two 2/1 white Insect creature tokens with flying.
+SVar:TrigToken:DB$ Token | TokenAmount$ 2 | TokenScript$ w_2_1_insect_flying
+Oracle:Impending 4—{2}{W}{W} (If you cast this spell for its impending cost, it enters with four time counters and isn't a creature until the last is removed. At the beginning of your end step, remove a time counter from it.)
+Whenever Overlord of the Mistmoors enters or attacks, create two 2/1 white Insect creature tokens with flying.
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/overlord_of_the_mistmoors.txt:10]: Bad line

File: forge-gui/res/cardsfolder/upcoming/veteran_survivor.txt [hunk 3]
```diff
@@ -0,0 +1,8 @@
+Name:Veteran Survivor
+Types:Creature Human Survivor
+PT:2/1
+T:Mode$ Phase | Phase$ Main2 | ValidPlayer$ You | IsPresent$ Card.Self+tapped | TriggerZones$ Battlefield | Execute$ TrigChangeZone | TriggerDescription$ Survival — At the beginning of your second main phase, if Veteran Survivor is tapped, exile up to one target card from a graveyard.
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/veteran_survivor.txt:4]: CARDNAME

File: forge-gui/res/cardsfolder/upcoming/kaito_bane_of_nightmares.txt [hunk 4]
```diff
@@ -0,0 +1,13 @@
+Name:Kaito, Bane of Nightmares
+ManaCost:2 U B
+Types:Legendary Planeswalker Kaito
+Loyalty:4
+K:Ninjutsu:1 U B
+S:Mode$ Continuous | Affected$ Permanent.Self+counters_GE1_LOYALTY | Condition$ PlayerTurn | AddType$ Creature & Ninja | SetPower$ 3 | SetToughness$ 4 | AddKeyword$ Hexproof | Description$ During your turn, as long as Kaito has one or more loyalty counters on him, he's a 3/4 Ninja creature and has hexproof.
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/kaito_bane_of_nightmares.txt:6]: NICKNAME

File: forge-gui/res/cardsfolder/upcoming/nowhere_to_run.txt [hunk 5]
```diff
@@ -0,0 +1,9 @@
+Name:Nowhere to Run
+ManaCost:1 B
+Types:Enchantment
+K:Flash
+T:Mode$ ChangesZone | Origin$ Any | Destination$ Battlefield | ValidCard$ Card.Self | Execute$ TrigPump | TriggerDescription$ When CARDNAME enters, target creature an opponent controls gets -3/-3 until end of turn.
+SVar:TrigPump:DB$ Pump | ValidTgts$ Creature.OppCtrl | TgtPrompt$ Select target creature an opponent controls | NumAtt$ -3 | NumDef$ -3 | IsCurse$ True
+S:Mode$ IgnoreHexproof | Activator$ You | ValidEntity$ Creature.OppCtrl | Description$ Creatures your opponents control can be the targets of spells and abilities as though they didn't have hexproof. Ward abilities of those creatures don't trigger.
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/nowhere_to_run.txt:7]: Activator wrong

File: forge-gui/res/cardsfolder/upcoming/valgavoth_terror_eater.txt [hunk 6]
```diff
@@ -0,0 +1,11 @@
+Name:Valgavoth, Terror Eater
+ManaCost:6 B B B
+Types:Legendary Creature Elder Demon
+PT:9/9
+K:Flying
+K:Lifelink
+K:Ward:Sac<3/Permanent.nonLand/nonland permanent>
+R:Event$ Moved | ActiveZones$ Battlefield | Destination$ Graveyard | ValidCard$ Card.nonToken+YouDontCtrl+OppOwn | ReplaceWith$ Exile | Description$ If a card you didn't control would be put into an opponent's graveyard from anywhere, exile it instead.
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/valgavoth_terror_eater.txt:8]: you need to test this
it might need ValidLKI$ instead

Comment (Fulgur14) [reply forge-gui/res/cardsfolder/upcoming/valgavoth_terror_eater.txt:8]: I tested Valgavoth, seemed to exile cards alright and when my stolen card died, it went normally to graveyard.

File: forge-gui/res/cardsfolder/upcoming/nowhere_to_run.txt (hunk 5 above)
Comment (tool4ever) [reply forge-gui/res/cardsfolder/upcoming/nowhere_to_run.txt]: just like `ValidCard$ Card` using `Activator$ Any` really serves no purpose
(other than script bloat + making the engine check for no reason)

**Review by tool4ever — DISMISSED** [review]

**Review by tool4ever — APPROVED** [review]

**Pull request merged**